### DISCUSSION TITLE
When will @fluentui/react-storybook-addon be released?

### DISCUSSION THREAD

**Discussion by sjwilczynski** [microsoft/fluentui#25257] [Q&A] [answered]
What are the plans for [@fluentui/react-storybook-addon package](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-storybook-addon/README.md)? We would love to reuse it for our storybook setup but it seems to not be released yet. Is there anything blocking its release? Being able to just select Fluent v9 theme inside story would be nice improvement.

**Comment by sjwilczynski** [comment, marked as answer]
@Hotell, @TristanWatanabe do you happen to know the answer?

**Reply by Hotell** [reply]
Hey @sjwilczynski ! 

ATM there is no ETA/priority to make it public. 

To give you more context:

As of today it contains some extend of public API that should not be public nor are conformant with storybook addon API surface.
We are currently refactoring bunch of storybook related stuff on our end so after that's done we might come to stage of polishing and releasing our official addon to npm. 

cheers

**Comment by goamn** [comment]
Any update on this? It's been nearly 1 year
